amber-mpris.pro

```diff
@@ -2,3 +2,10 @@ TEMPLATE = subdirs
 declarative.depends = src
 src.depends = qtdbusextended
 SUBDIRS = src declarative qtdbusextended
+
+CONFIG += mer-qdoc-template
```

Comment:
sailfish-qdoc-template please :)

Reply:
And move this to doc/doc.pro. CONFIG on TEMPLATE=subdirs would work by accident.

Reply:
Done.